src/App.js

```diff
@@ -14,11 +15,69 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  render() {
+    const allGoods = [...goodsFromServer];
+    const { selectedGood } = this.state;
+
+    return (
+      <div className="App">
+        {selectedGood ? (
+          <h1 className="title">
+            <span className="title__text">
+              {`${selectedGood} is selected`}
+            </span>
+            <button
+              className="remove-btn"
+              type="button"
+              onClick={() => {
+                this.setState({
+                  selectedGood: null,
+                });
+              }}
+            >
+              x
+            </button>
+          </h1>
+        ) : (
+          <h1>No goods selected</h1>
+        )}
+        <ul className="list">
+          {allGoods.map(good => (
+            <li
+              className={selectedGood === good
+                ? 'active'
+                : ''
+              }
+              key={good}
+            >
+              <span>
+                {good}
+              </span>
+              {selectedGood === good ? (''
+              ) : (
+                <button
```

Comment:
```suggestion
              {selectedGood !== good && (
                <button
```
Let's use conditional rendering syntax